persist/sqlite/consensus_refactored_test.go

```diff
@@ -2558,6 +2558,70 @@ func TestHostScan(t *testing.T) {
 	assertHost(hosts[0].PublicKey, host1)
 }
 
+func TestEventPayoutContract(t *testing.T) {
```

Comment:
`nit` instead of leaving a comment inside of the test, it's probably a bit cleaner to have a docstring that looks like  `// TestEventPayoutContract is a regression test that covers a bug in ...`